elytron/src/main/java/org/wildfly/extension/elytron/RealmParser.java

```diff
@@ -137,11 +154,11 @@
             .addChild(customRealmParser)
             .addChild(customModifiableRealmParser)
             .addChild(identityRealmParser)
-            .addChild(jdbcRealmParser_7_0)
+            .addChild(jdbcRealmParser_11_0)
```

Comment:
Some more work is needed in this parser. This existing block should be copied to a new realmParser variable and then a new variable should be introduced that contains these changes.